docs/specs/short-read-alignment-vector.md

```diff
@@ -0,0 +1,215 @@
+# Short read alignment (vector) pipeline specification
+
+* Version: 1.1.1
+* Authors: Alistair Miles, Jim Stalker
+
+This document specifies a protocol for alignment of short sequence
+reads, intended for use with mosquito specimens.
+
+
+## Assumptions
+
+* The pipeline will be launched to align sequence reads from one or
+  more samples as a batch.
+
+* Any demultiplexing of multiple samples within a single sequencing
+  lane will have been resolved upstream of this pipeline. I.e., the
+  pipeline will run from demultiplexed sequence reads. The term
+  "lanelet" is used here to mean data from a single lane of sequencing
+  from a single sample.
+
+* Each sample may have data from multiple sequencing runs (lanes), and
+  the number of lanelets may vary between samples. I.e., the pipeline
+  cannot assume a fixed number of lanelets per sample.
+
+* The complete set of input data will be known ahead of time, before
+  the pipeline is launched for a given batch of samples. I.e., there
+  is no need to consider any case where additional lanelets may arrive
+  at a later time.
+
+* Sequencing will be from Illumina technology, using paired-end
+  sequencing. Mosquitoes will be sequenced individually, i.e., each
+  mosquito sample will be used to create a separate DNA library, and
+  libraries will not be pooled. Each individual mosquito will be
+  sequenced to high coverage (target ~30X). Read length may vary
+  between sequencing runs, i.e., the pipeline cannot assume the same
+  read length across all lanelets.
+
+* Trimming of adapter sequences, if necessary, will have been
+  performed prior to running this pipeline.
+
+
+## Inputs
+
+* Reference sequence - The reference genome sequence to align to, in
+  FASTA format.
+
+* Sequence reads - For each lanelet, a pair of files containing
+  sequence reads, with one file containing forward reads and one file
+  containing reverse reads, in FASTQ format.
+
+* Lanelet manifest - A tab delimited file with one row per
+  lanelet. Mandatory columns are as follows, may contain any other
+  columns:
+
+    * `read1_path` - URL or local filesystem path to FASTQ file
+      containing forward reads.
+
+    * `read2_path` - URL or local filesystem path to FASTQ file
+      containing reverse reads.
+
+    * `sample_id` - Identifier for the sample that was sequenced.
+
+
+## Outputs
+
+* One analysis-ready BAM file for each sample.
+
+* samtools stats output for each sample.
+
+* GATK CallableLoci output for each sample.
```

Comment:
I'm not as familiar with this tool, does it output VCF? gVCF?

Reply:
It outputs a TSV of statistics on how many bases in a sample were callable.  The definition of callable is configurable and a function of mapping quality, depth, etc.  The tool is no longer available in GATK 4.x, as far as I know.

Reply:
IIRC it also outputs a text file which provides a classification of all regions of the genome, something a bit like a BED file, although need to check that.